tests/foreman/ui/test_subscription.py

```diff
@@ -551,3 +551,21 @@ def test_positive_candlepin_events_processed_by_STOMP(
         response = entities.Ping().search_json()['services']['candlepin_events']
         assert response['status'] == 'ok'
         assert '0 Failed' in response['message']
+
+
+def test_positive_prepare_for_sca_only_subscription(target_sat, function_entitlement_manifest_org):
+    """Verify that the Subcsription page notifies users that Simple Content Access
+        will be required for all organizations in Satellite 6.16
+
+    :id: cb6fdfdd-04ee-4acb-9460-c78556cef11e
+
+    :expectedresults: The Subscription page notifies users that Simple Content Access will
+        be required for all organizations in Satellite 6.16
+    """
+    with target_sat.ui_session() as session:
+        session.organization.select(function_entitlement_manifest_org.name)
+        sca_alert = session.subscription.sca_alert()
```

Comment:
@ColeHiggins2 PRT failed with `AttributeError: 'SubscriptionEntity' object has no attribute 'sca_alert'`, I think you'll need to decorate sca_alert with `@property` to fix this in airgun. I think it was missed in your original master branch PR as you didn't run the PRT and it was merged w/o PRT

Reply:
Just got a fix for this in airgun, lets rerun prt now

Reply:
just for future ref, PR fixing this https://github.com/SatelliteQE/airgun/pull/1123